tests/test_physics_3d.cpp

```diff
@@ -93,7 +93,7 @@ class TestPhysics3DMainLoop : public MainLoop {
 		ps->body_set_param(body, PhysicsServer3D::BODY_PARAM_BOUNCE, 0.0);
 		//todo set space
 		ps->body_add_shape(body, type_shape_map[p_shape]);
-		ps->body_set_force_integration_callback(body, this, "body_changed_transform", mesh_instance);
+		ps->body_set_force_integration_callback(body, callable_mp(this, &TestPhysics3DMainLoop::body_changed_transform), mesh_instance);
```

Comment:
You can also remove `ClassDB::bind_method("body_changed_transform", &TestPhysics3DMainLoop::body_changed_transform);` (and the whole `_bind_method` override).